connectors/airwatch/src/main/java/com/vmware/connectors/airwatch/AirWatchController.java

```diff
@@ -216,6 +233,10 @@ public AirWatchController(WebClient rest, CardTextAccessor cardTextAccessor,
                 .setHeader(cardTextAccessor.getHeader(locale, appName))
                 .setBody(cardBodyBuilder.build())
                 .addAction(appInstallActionBuilder.build());
+
+        // Set Image url.
+        CommonUtils.buildConnectorImageUrl(cardBuilder, request);
```

Comment:
We don't want to set this currently.

Notice that AirWatch connector does not have an image right now:
* https://github.com/vmware/connectors-workspace-one/tree/master/connectors/airwatch/src/main/resources/static

Reply:
I have removed ```buildConnectorImageUrl``` from the Airwatch connector.